addons/gdUnit3/bin/GdUnitCopyLog.gd

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env -S godot -s
+extends MainLoop
+
+const NO_LOG_TEMPLATE = """
+<!DOCTYPE html>
+<html>
+<head>
+	<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+	<meta http-equiv="x-ua-compatible" content="IE=edge"/>
+	<title>Logging</title>
+	<link href="css/style.css" rel="stylesheet" type="text/css"/>
+</head>
+<body>
+	<div>
+		<h1>No logging available!</h1>
+		</br>
+		<p>For logging to occur, you must check Enable File Logging in Project Settings.</p>
+		<p>You can enable Logging <b>Project Settings</b> > <b>Logging</b> > <b>File Logging</b> > <b>Enable File Logging</b> in the Project Settings.</p>
+	</div>
+</body>
+"""
+
+#warning-ignore-all:return_value_discarded
+var _cmd_options: = CmdOptions.new([
+			CmdOption.new("-rd, --report-directory", "-rd <directory>", "Specifies the output directory in which the reports are to be written. The default is res://reports/.", TYPE_STRING, true),
+])
+
+var _report_root_path :String
+
+func _init():
+	_report_root_path = GdUnitTools.current_dir() + "reports"
+
+func _idle(_delta):
+	# check if reports exists
+	if not reports_available():
+		prints("no reports found")
+		return true	
+	# scan for latest report path
+	var iteration = GdUnitTools.find_last_path_index(_report_root_path, GdUnitHtmlReport.REPORT_DIR_PREFIX)
+	var report_path = "%s/%s%d" % [_report_root_path, GdUnitHtmlReport.REPORT_DIR_PREFIX, iteration]
+	
+	# only process if godot logging is enabled
+	if not GdUnitSettings.is_log_enabled():
+		_patch_report(report_path, "")
+		return true
+	
+	# parse possible custom report path, 
+	var cmd_parser := CmdArgumentParser.new(_cmd_options, "GdUnitCmdTool.gd")
+	# ignore erros and exit quitly
+	if cmd_parser.parse(OS.get_cmdline_args(), true).is_error():
+		return true
+	CmdCommandHandler.new(_cmd_options).register_cb("-rd", funcref(self, "set_report_directory"))
+	
+	# scan for latest godot log and copy to report
+	var godot_log := _scan_latest_godot_log()
+	var result := _copy_and_pach(godot_log, report_path)
+	if result.is_error():
+		push_error(result.error_message())
+		return true
+	
+	_patch_report(report_path, godot_log)
+	return true
+
+func set_report_directory(path :String) -> void:
+	_report_root_path = path
+
+func _scan_latest_godot_log() -> String:
+	var path := GdUnitSettings.get_log_path().get_base_dir()
+	var files_sorted := Array()
+	for file in GdUnitTools.scan_dir(path):
+		var file_name := "%s/%s" % [path,file]
+		files_sorted.append(file_name)
+	# sort by modification time
```

Comment:
add better doc, 
it is sorting primary by the file name but the name contains the time stamp where is resulting into sort by time stamp at the end